src/core/lwip/tcp_out.c

```diff
@@ -738,6 +742,122 @@ err_t tcp_write(struct tcp_pcb *pcb, const void *arg, u32_t len, u16_t apiflags,
     return ERR_MEM;
 }
 
+err_t tcp_write_express(struct tcp_pcb *pcb, const void *arg, u32_t len, pbuf_desc *desc)
+{
+    struct pbuf *p;
+    struct tcp_seg *seg = NULL, *prev_seg = NULL, *queue = NULL;
+    u32_t pos = 0; /* position in 'arg' data */
+    u8_t optflags = TF_SEG_OPTS_ZEROCOPY;
+    const u16_t mss_local = lwip_zc_tx_size;
+    u16_t seglen;
+    u16_t queuelen = 0;
+
+    if (len < pcb->mss) {
+        const int byte_queued = pcb->snd_nxt - pcb->lastack;
+        pcb->snd_sml_add = (pcb->unacked ? pcb->unacked->len : 0) + byte_queued;
+    }
+
+    /*
+     * TCP segmentation is done in three phases with increasing complexity:
+     *
+     * 1. Copy data directly into an oversized pbuf.
```

Comment:
Please update comment to represent current functionality.

Reply:
Done